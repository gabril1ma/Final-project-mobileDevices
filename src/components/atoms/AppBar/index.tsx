import React from "react";
import { View, ViewProps } from "react-native";

interface IAppBar extends ViewProps {}
const AppBar = ({ children, ...props }: IAppBar) => {

  return (
    <View 
      {...props}>{children}
      
    </View>
  );
};

export default AppBar;
