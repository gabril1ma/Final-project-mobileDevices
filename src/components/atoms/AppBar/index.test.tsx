import { create, ReactTestRendererJSON } from "react-test-renderer";
import AppBar from ".";

export type TTreeRenderer = ReactTestRendererJSON | ReactTestRendererJSON[];

describe("<AppBar />", () => {
  let tree: TTreeRenderer;
  let testTree: ReactTestRendererJSON;

  beforeEach(() => {
    tree = create(<AppBar />).toJSON();

    testTree = tree[0] ?? tree;
  });

  it("should render component", () => {
    expect(testTree).toMatchSnapshot();
  });
});
