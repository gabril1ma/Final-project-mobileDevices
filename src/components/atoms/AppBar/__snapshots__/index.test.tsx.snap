// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<AppBar /> should render component 1`] = `<View />`;
